apps/wpcom-block-editor/package.json

```diff
@@ -37,10 +37,12 @@
 		"@wordpress/hooks": "^3.2.2",
 		"@wordpress/i18n": "^4.2.4",
 		"@wordpress/icons": "^6.1.1",
+		"@wordpress/interface": "^4.4.0",
```

Comment:
Let's use `@wordpress/interface@4.1.15` for now -- that's in line with the WP monorepo packages we use in Calypso currently. If you look here:

https://github.com/Automattic/wp-calypso/blob/de6144be27aefaeabe7593e7aac0682f557afa8b/.yarnrc.yml#L231

We have added overrides to resolve the problem for this version of the interface package. We'll update all the WordPress packages at the same time in [this PR](https://github.com/Automattic/wp-calypso/pull/60161), which should hopefully be merged in the next week or two.